styles/index.css

```diff
@@ -0,0 +1,10 @@
+body {
+    background-image: url("/images/jay-mantri-TFyi0QOx08c-unsplash.jpg");
+    background-size: cover;
+    background-color: blue;
+
+    height: 100vh;
```

Comment:
This is usually a nice way to get a bg image to cover the body, but we often need to use an additional rule to make sure there's no stray margin/padding lurking around the body, otherwise this can make the body spill beyond the end of the window and add unnecessary scroll bars.